tests/python/gaia-ui-tests/gaiatest/tests/functional/ftu/test_ftu_status_bar.py

```diff
@@ -0,0 +1,34 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.ftu.app import Ftu
+from gaiatest.apps.system.regions.status_bar import StatusBar
+
+
+class TestFtu(GaiaTestCase):
+
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.ftu = Ftu(self.marionette)
+        self.ftu.launch()
+
+    def test_ftu_status_bar(self):
+        """
+        https://moztrap.mozilla.org/manage/cases/?filter-id=6119
```

Comment:
So I would precise that here in case of a new reader is not aware of `test_ftu_skip_tour.py`